### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by ahzamir**
### In this pull request, I have added `app.rb` and `main.rb` files to create my app UI based on these requirements:

- I created a `app.rb` file that will serve as my console app entry-point. It has methods that do the following:
  - List all books.
  - List all people.
  - Create a person (teacher or student).
  - Create a book.
  - Create a rental.
  - List all rentals for a given person's id.
- In My `main.rb` I defined the entry point and it has a method called main that is invoked at the end of my file. This method does the following:
  - Present the user with a list of options to perform.
  - Lets users choose an option.
  - If needed, ask for parameters for the option.
  - Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    book = Book.new(title, author)
+    @books.push(book)
+    puts 'A book created successfully 📘👋'
+    run
+  end
+
+  def create_rental
+    puts 'Select a book from following list by number (not ID):'
+    @books.each_with_index do |book, idx|
+      puts "#{idx}) Title: '#{book.title}', Author: '#{book.author}'"
```
Comment (KDlamini) [inline app.rb:143]: When I create a rental I get an error.

image

- Kindly make sure that users can create rentals successfully and are able to list all rentals from a specific user using id.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @ahzamir 🙋‍♀️,  

Excellent work! 💯 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞
 ✔️  Good UI and usability
✔️  Good PR title and description
✔️  Correct use of Gitflow 
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (ahzamir) [reply app.rb:143]: @KDlamini  Thank you very much resolved. 👍😊

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @ahzamir,

Good job on fixing the changes asked by the previous reviewer on creating a rental 👍 

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:
You did well so far on the following

✅ Correctly set up Gitflow
✅ No linter error
✅ README looks great
✅ Good commit messages
✅ Good descriptive PR

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by ahzamir** [comment]
@KDlamini  @AmanduKamara Thank you very much for your kind review and feedback. 😊👍

**Pull request merged**